src/ui/components/shared/Login/Login.tsx

```diff
@@ -209,13 +210,18 @@ export function LoginLink({
 }
 
 export default function Login({ returnToPath = "" }: { returnToPath?: string }) {
-  const { loginWithRedirect, error } = useAuth0();
+  const { loginWithRedirect, error, user } = useAuth0();
+  const router = useRouter();
   const [sso, setSSO] = useState(false);
 
   if (returnToPath.startsWith("/login")) {
     returnToPath = "/";
   }
 
+  if (user) {
```

Comment:
We should add `&& returnToPath` here since it could be unset and the redirect wouldn't make much sense